src/main/java/bio/terra/service/filedata/flight/ingest/FileIngestBulkFlight.java

```diff
@@ -67,6 +72,8 @@ public FileIngestBulkFlight(FlightMap inputParameters, Object applicationContext
     UUID datasetUuid = UUID.fromString(datasetId);
     Dataset dataset = datasetService.retrieve(datasetUuid);
 
+    var platform = CloudPlatformWrapper.of(dataset.getDatasetSummary().getStorageCloudPlatform());
```

Comment:
Should this be `var platform =
        CloudPlatformWrapper.of(
            inputParameters.get(JobMapKeys.CLOUD_PLATFORM.getKeyName(), CloudPlatform.class));` to be consistent with the other file ingest flights?

Reply:
Yeah good call.  I was trying to get the test runner test to pass and this one slipped...I'll update